src/libaktualizr/uptane/uptane_test.cc

```diff
@@ -477,6 +477,7 @@ TEST(Uptane, ProvisionOnServer) {
 TEST(Uptane, FsToSqlFull) {
   TemporaryDirectory temp_dir;
   Utils::copyDir("tests/test_data/prov", temp_dir.Path());
+  chmod(temp_dir.Path().c_str(), S_IRWXU);
```

Comment:
Why don't we just do this automatically in `copyDir`? I think it's safe to say that we always want to enforce the same specific permissions.

Reply:
I think it's better to do it explicitly. Also, usual `copy` behavior is to copy permissions from the source. So if we replace `Utils::copyDir` with `copy_directory` from boost or C++14 this line will be needed.

Reply:
Fair enough, that sounds reasonable.